playground/src/components/header/index.tsx

```diff
@@ -16,17 +15,16 @@ interface LinkInfo {
 }
 
 const links: LinkInfo[] = [
-  { icon: faClone, label: "Endpoints", to: "/playground" },
-  { icon: faCog, label: "Configuration", to: "/playground/configuration" },
+  { icon: faClone, label: "Endpoints", to: "/playground/" },
+  { icon: faCog, label: "Configuration", to: "/playground/configuration/" },
 ];
 
 function Header() {
-  const { routerStore } = React.useContext(RootStore);
+  const { pathname } = useLocation();
 
-  const activePath = routerStore.location.pathname;
   const linkElements = links.map(l => (
     <Link key={l.to} to={l.to}>
-      <div className={classnames(s.link, l.to === activePath && s.active)}>
+      <div className={classnames(s.link, l.to === pathname ? s.active : "")}>
```

Comment:
Think this div is unnecessary, links usually turn into the active state automatically when the URL matches its href. You can just style a:active 😉